mypyc/genops.py

```diff
@@ -2297,6 +2342,17 @@ def unbox_or_cast(self, src: Value, target_type: RType, line: int) -> Value:
     def box_expr(self, expr: Expression) -> Value:
         return self.box(self.accept(expr))
 
+    def make_dict(self, keys: List[Value], values: List[Value], line: int) -> Value:
```

Comment:
It would be cleaner to take a `List[Tuple[Value, Value]]` of dict entries, I think. Parallel arrays are usually not the cleanest way to do things. (All of the parallel arrays in the mypy AST included, really.)

Reply:
Changed.